sql/2024/cookies/0_create_desktop_cookies.sql

```diff
@@ -0,0 +1,39 @@
+-- Create an intermediate table containing all cookies that were set during the
+-- <DATE> crawl on <CLIENT> when visiting sites of rank <= <RANK>. This table
+-- can then be reused in consecutive queries without having to reextract the
+-- data every time
+-- Export the table as httparchive.almanac.DATE_CLIENT_RANK_cookies
+
+WITH intermediate_cookie AS (
+  SELECT
+    page,
```

Comment:
Do we want this to restrict this to just root_pages? Or sum up cookies across both home and secondary pages for the same `root_page` so we can count sites using cookies?

I see you use `NET.HOST(page)` later on but `root_page` would perhaps be better and allow this filtering up front.

Reply:
I tried to be as general as possible when extracting the cookies table, and I did not know up front if we were going to only stick to `root_page` or no for this year's queries, same for in the future.